src/swap/useSwapQuote.ts

```diff
@@ -52,10 +53,13 @@ const useSwapQuote = () => {
     }
 
     const swapAmountParam = updatedField === Field.FROM ? 'sellAmount' : 'buyAmount'
-    const requestUrl = `${networkConfig.approveSwapUrl}?buyToken=${toToken.address}&sellToken=${
-      fromToken.address
-    }&${swapAmountParam}=${swapAmountInWei.toString().split('.')[0]}&userAddress=${walletAddress}`
-
+    const params = {
+      buyToken: toToken.address,
+      sellToken: fromToken.address,
+      [swapAmountParam]: swapAmountInWei.toString().split('.')[0],
+      userAddress: walletAddress,
+    }
+    const requestUrl = `${networkConfig.approveSwapUrl}?${qs.stringify(params)}`
```

Comment:
i'm not really convinced that the readability lift we get here is worth the extra 2 dependencies.

if we want to avoid the long string template literal, i'd suggest converting the params object to url params ourselves. something like
```
const requestUrl = `${networkConfig.approveSwapUrl}?${new URLSearchParams(Object.entries(params)).toString()}`
```

Reply:
Removed and replaced with something similar in https://github.com/valora-inc/wallet/pull/2767/commits/70a84c7f3df4ca51bc90fa976497ba6494dbf1f4!